onnxscript/rewriter/rules/common/_remove_zero_bias_test.py

```diff
@@ -0,0 +1,431 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Tests for removing zero bias from Conv and related operations."""
+
+import unittest
+from typing import Optional
+
+import onnx_ir as ir
+
+from onnxscript.rewriter import testing
+from onnxscript.rewriter.rules.common._remove_zero_bias import (
+    remove_zero_bias_from_conv_rule,
+    remove_zero_bias_from_conv_transpose_rule,
+    remove_zero_bias_from_gemm_rule,
+    remove_zero_bias_from_qlinear_conv_rule,
+)
+
+
+def _apply_rule_and_check_optimization(
```

Comment:
This test is a little too complicated to maintain. Please check other tests in the directory and simplify.